src/main/java/duke/Command/FindCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.Command;
+
+import duke.*;
```

Comment:
I think you should import package by package, for example `import duke.TaskList` rather than using `import duke.*`